src/components/apps/launch/AppLaunchForm.js

```diff
@@ -48,100 +51,108 @@ import {
 
 import {
     Container,
-    BottomNavigation,
-    BottomNavigationAction,
+    Button,
     Hidden,
     Stepper,
     Step,
     StepButton,
     StepLabel,
+    Toolbar,
     Typography,
     makeStyles,
+    useMediaQuery,
+    useTheme,
 } from "@material-ui/core";
 
 import { ArrowBack, ArrowForward, PlayArrow, Save } from "@material-ui/icons";
 
 const useStyles = makeStyles(styles);
 
+// sonora appbar height.
+//Stepper height, appinfo height and bottom nav height is calculated dynamically.
+const NAVIGATION_BAR_HEIGHT = 105;
+const MOBILE_NAVIGATION_BAR_HEIGHT = 75;
```

Comment:
Probably not for this PR, but a later PR - the Sonora app bar height isn't hardcoded in `_app` or `CyVerseAppBar`, it's also figured out dynamically.  I'd like us to keep with that pattern if MUI ever decides to change these heights.

Reply:
Ya we have to figure out how to pass down the ref between appbar  and app launch.